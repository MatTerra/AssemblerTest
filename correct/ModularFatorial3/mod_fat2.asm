mod_fat: begin
mod_fim: extern
aux: extern
n: extern
fim: extern
public one
public mod_fat
section data
ONE: CONST 1
section text
fat: sub ONE
JMPZ fim
STORE AUX
MUL N
STORE N
LOAD auX
JMP MOD_FIM
end
