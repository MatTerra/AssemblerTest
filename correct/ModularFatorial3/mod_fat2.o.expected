H: mod_fat2
H: 15
H: 010101010101010
H: U aux 5 11
H: U fim 3
H: U mod_fim 13
H: U n 7 9
H: D mod_fat 0
H: D one 14
T: 02 14 08 00 11 00 03 00 11 00 10 00 05 00 1
